_docs/reference/release_notes.md

```diff
@@ -52,6 +52,83 @@ Due to the nature the KV Store change there is no upgrade method from 0.7.x to
 new parameters (`CLUSTER_ID` and `INITIAL_CLUSTER`).  Note that `CLUSTER_ID` and
 `INITIAL_CLUSTER` have been replaced by `JOIN` in 0.9.x onwards.
 
+## 0.9.1
+
+The 0.9.1 release includes a fix for the `mount deadline exceeded` error, where
+a volume mount fails after a long delay.
+
+The logging susbsystem has also been overhauled to reduce noise and to allow
+filtering messages by log level and component.  We expect to build on this in
+upcoming releases to further improve diagnostics.
+
+### New
+
+- Failure modes can be specified to control behaviour in specific failure
+scenarios.  Full documentation to be added to the docs site soon.
+
+  - `hard` will enforce the desired number of replicas and take a volume offline
+    if the replica count is not met.  In practice, the volume should only go
+    offline if there isn't a suitable node to place a new replica on.  For
+    example, if you have configured 2 replicas in a 3 node cluster and a node
+    goes offline.
+  - `alwayson` will optimise for availability and will keep a volume online even
+    if all replicas have failed.
+  - `soft` is set by default, and will only take a volume offline if the number
+    of replicas falls below the `SoftFailureTolerance` (default: number of
```

Comment:
SoftFailureTolerance is called Failure tolerance in https://github.com/storageos/storageos.github.io/pull/105/files. We should be consistent - probably Failure Tolerance as that's the label.